src/Orchard.Web/Modules/Orchard.AntiSpam/ViewModels/ReCaptchaPartEditViewModel.cs

```diff
@@ -1,10 +1,15 @@
-﻿namespace Orchard.AntiSpam.ViewModels {
+﻿using Newtonsoft.Json;
+
+namespace Orchard.AntiSpam.ViewModels {
     public class ReCaptchaPartEditViewModel {
         public string PublicKey { get; set; }
     }
 
-    public class ReCaptchaPartSubmitViewModel {
-        public string recaptcha_challenge_field { get; set; }
-        public string recaptcha_response_field { get; set; }
+    public class ReCaptchaPartResponseModel {
+        [JsonProperty("success")]
+        public string Success { get; set; }

```

Comment:
This does need to be changed back to a boolean. sfmskywalker has already done this.